### DISCUSSION TITLE
postgres - database error: error with configurationn: invalid port number

### DISCUSSION THREAD

**Discussion by zywsky** [quickwit-oss/quickwit#5520] [General]
When I am trying below kind of postgres url as the metastore_uri value, and met "database error: error with configurationn: invalid port number" error.

postgres://host1:123,host2:456/mydb
https://github.com/quickwit-oss/quickwit/blob/1be69915f29c40f51350c2f818563109edef87c4/docs/configuration/metastore-config.md?plain=1#L38 

Do you know the reason? seems it's not supported for this kind of configuration.

**Comment by guilload** [comment]
I don't think we support Postgres URIs with multiple hosts. `sqlx`, the crate we use to interact with Postgres, does not support it according to this open [issue](https://github.com/launchbadge/sqlx/issues/179). I'll fix our documentation.
